delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Organizations.scala

```diff
@@ -105,6 +107,14 @@ trait Organizations {
       params: OrganizationSearchParams = OrganizationSearchParams.none
   ): UIO[UnscoredSearchResults[OrganizationResource]]
 
+  /**
+    * A non terminating stream of events for organizations. After emitting all known events it sleeps until new events
+    * are recorded.
+    *
+    * @param offset the last seen event offset; it will not be emitted by the stream
+    */
+  def events(offset: Offset = NoOffset): Stream[Task, Envelope[OrganizationEvent]]
```

Comment:
should we also add `currentEvents()` like in permissions?

Reply:
I haven't added it in Realms either

Reply:
Not yet.